bedrock/firefox/templates/firefox/os/index-new.html

```diff
@@ -0,0 +1,241 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/base-resp.html" %}
+
+{% add_lang_files "firefox/os/devices" %}
+
+{% block page_title_prefix %}{% endblock %}
+{% block page_title %}{{_('Firefox OS — The Adaptive Phone — Great Smartphone Features, Apps and More')}}{% endblock %}
+{% block page_desc %}{{_('Firefox OS smartphones give you great adaptive features and apps that let you live every moment to its fullest and build a brighter future for the Web.')}}{% endblock %}
+
+{% block body_id %}firefox-os{% endblock %}
+{% block body_class %}firefox-os{% endblock %}
+
+{# Do not load CSS bundle specified in base template (duplicate CSS). #}
+{% block site_css %}{% endblock %}
+
+{% block extrahead %}
+  {{ super() }}
+  {% stylesheet 'firefox_os_new' %}
+<!--[if lt IE 9]>
+  {% stylesheet 'firefox_os_ie' %}
+<![endif]-->
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block site_header_unwrapped %}
+  {% include 'firefox/includes/family-nav.html' %}
+{% endblock %}
+
+{% block content %}
+  <main role="main">
+    <section class="fxos-hero-space">
+      <div class="content-wrapper">
+        <div class="phone">
+          <div class="fxos-cta">
+            <h1><span>{{_('Firefox <span>OS<span>')}}</span></h1>
```

Comment:
Yeah it's nothing major but slightly unfortunate, since it's invalidating the markup. @flodolo anything we can do here without causing fallout? :smile: